openslides_backend/action/motion_statute_paragraph/create_update_delete.py

```diff
@@ -0,0 +1,20 @@
+from ...models.motion_statute_paragraph import MotionStatuteParagraph
+from ..action import register_action_set
+from ..action_set import ActionSet
+from ..default_schema import DefaultSchema
+
+
+@register_action_set("motion_statute_paragraph")
+class MotionStatuteParagraphActionSet(ActionSet):
+    """
+    Actions to create, update and delete motion statute paragraph.
+    """
+
+    model = MotionStatuteParagraph()
+    create_schema = DefaultSchema(MotionStatuteParagraph()).get_create_schema(
+        properties=["title", "text"], required_properties=["title"],
+    )
+    update_schema = DefaultSchema(MotionStatuteParagraph()).get_update_schema(
+        properties=["title"]
+    )
+    delete_schema = DefaultSchema(MotionStatuteParagraph()).get_delete_schema()
```

Comment:
Please see https://github.com/OpenSlides/openslides-backend/blob/master/openslides_backend/action/agenda_item/create.py#L58 how to set the default for weight field in create action. If this does not fit with the ActionSet because you can not patch the correct `update_instance` method (only for create and not for update or delete), we should add some small logic to the model fields to set the default overthere.

Reply:
Created a Create-Action, which sets the weight default and add it to the action set.